src/BaseClient.php

```diff
@@ -394,7 +394,6 @@ protected function defaultRequestOptions()
     {
         return [
             'timeout' => 30,
-            'sslVerifyPeer' => false,
```

Comment:
the same exists in BaseOAuth, and OpenId classes: 

- https://github.com/yiisoft/yii2-authclient/blob/d34ee5d924bfb151d6b4028c57c57dd4b18aedf9/src/OpenId.php#L283
- https://github.com/yiisoft/yii2-authclient/blob/d34ee5d924bfb151d6b4028c57c57dd4b18aedf9/src/BaseOAuth.php#L191